piplicenses.py

```diff
@@ -78,6 +50,11 @@ def get_installed_distributions():
     from prettytable import NONE as RULE_NONE
     PTABLE = False
 
+if sys.version_info < (3, 8):
+    import importlib_metadata
+else:
+    from importlib import metadata as importlib_metadata
+
```

Comment:
> Note that, for python 3.7, the proposed change implies that `importlib_metadata` (Apache license) is installed as a `pip-licenses` dependency. I think this dependency should be added to `SYSTEM_PACKAGES` just like `PTable`/`prettytable` ?

We are currently trying to stop using `PTable` and move to `prettytable`. #82

> In the current version of prettytable 2.5.0, the number of dependent packages has increased.
> 
> https://github.com/jazzband/prettytable/blob/2.5.0/setup.cfg
> ```
> [options]
> install_requires =
>     wcwidth
>     importlib-metadata;python_version < '3.8'
> ```

To complete the migration, we thought we would add `wcwidth` / `importlib-metadata` to `SYSTEM_PACKAGES`.

Alternatively, we could choose to upgrade as pip-licenses 4.0 (allow broken changes), stop supporting Python 3.7, and not depend on `importlib-metadata`.

@b-kamphorst @johnthagen I would like to know if you have an opinion.

Reply:
@johnthagen Given the alternatives, I would vote to keep things simple and drop support for 3.7.

This will help `pip` install an older version of `pip-licenses` for those still using 3.7 for a short time:

https://github.com/raimon49/pip-licenses/blob/12d5d5f25d6347ec671b723f548a43a87d84c7fd/setup.cfg#L23

Reply:
I'm not a fan per se -- python 3.7 is still [officially supported](https://www.python.org/downloads/) for 8 more months. Having said that, given the stability of `pip-licenses` over the last years, I'd like to think that the current version will suffice for those 8 months.

Just to be sure: shouldn't we update the line `python_requires = ~=3.7` to `python_requires = ~=3.8` and drop the classifier for Python 3.7 if we want pip to do the right thing (e.g. install `pip-licenses==3.5.5` instead of `4.0.0`)?

Reply:
> Just to be sure: shouldn't we update the line python_requires = ~=3.7 to python_requires = ~=3.8 and drop the classifier for Python 3.7 if we want pip to do the right thing 

Yes, exactly.